apps/hash-ai-worker-ts/src/activities/shared/activity-logger.ts

```diff
@@ -23,15 +23,20 @@ const log = (
   const logMessage = `[Request ${requestId} – ${new Date().toISOString()}] ${message}`;
   const logFolderPath = path.join(__dirname, "logs");
 
-  if (process.env.NODE_ENV === "development") {
+  if (["test", "development"].includes(process.env.NODE_ENV ?? "")) {
     if (!fs.existsSync(logFolderPath)) {
       fs.mkdirSync(logFolderPath);
     }
     const logFilePath = path.join(logFolderPath, `${requestId}.log`);
     fs.appendFileSync(logFilePath, `${logMessage}\n`);
   }
 
-  logToConsole[level](logMessage);
+  if (process.env.NODE_ENV === "test") {
+    // eslint-disable-next-line no-console
+    console.log(logMessage);
```

Comment:
Why not use `logToConsole` here?

Reply:
Because it doesn't actually log anything when I run the test via `vitest`, not entirely sure why haven't spent a lot of time trying to fix this.

Reply:
Fair play, don't worry about it then